test/testcafe/spec/IdentifyRegistration_spec.js

```diff
@@ -154,36 +152,86 @@ test.requestHooks(enrollProfileErrorMock)('should show email field validation er
       }
     }
   });
-
 });
 
-
-test.requestHooks(enrollProfileFinishMock)('should show terminal screen after registration', async t => {
+test.requestHooks(mock)('should show terminal screen after registration', async t => {
   const registrationPage = await setup(t);
+  await verifyRegistrationPageEvent(t);
+
+  // click register button
+  await registrationPage.fillFirstNameField('abc');
+  await registrationPage.fillLastNameField('xyz');
+  await registrationPage.fillEmailField('foo@ex.com');
+  await registrationPage.clickRegisterButton();
+
+  // show successful terminal view and fires after render event
+  await t.expect(registrationPage.getTerminalContent()).eql(
+    'An activation email has been sent to john@gmail.com. Follow instructions in the email to finish creating your account'
+  );
+
+  const { log: log2 } = await t.getBrowserConsoleMessages();
```

Comment:
Can we use shared util `checkConsoleMessages` here ? https://github.com/okta/okta-signin-widget/blob/ec377eeb098304af04aeb93427b4538b7503f76c/test/testcafe/spec/Introspect_spec.js#L24